modules/loki-deployment-sizing.adoc

```diff
@@ -13,61 +13,71 @@ endif::[]
 Sizing for Loki follows the format of `<N>x.<size>` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
 
 .Loki sizing
-[cols="1h,3*",options="header"]
+[cols="1h,4*",options="header"]
 |===
 |
+|1x.demo
 |1x.extra-small
 |1x.small
 |1x.medium
 
 |Data transfer
+|Demo use only
 |100GB/day
 |500GB/day
 |2TB/day
 
 |Queries per second (QPS)
+|Demo use only
 |1-25 QPS at 200ms
 |25-50 QPS at 200ms
 |25-75 QPS at 200ms
 
 |Replication factor
+|None
 |2
 |2
 |2
 
 |Total CPU requests
+|None
 |14 vCPUs
 |34 vCPUs
 |54 vCPUs
 
 ifdef::restricted[]
 |Total CPU requests if using the ruler
+|None
 |16 vCPUs
 |42 vCPUs
 |70 vCPUs
 endif::restricted[]
 
 |Total memory requests
+|None
 |31Gi
 |67Gi
 |139Gi
 
 ifdef::restricted[]
 |Total memory requests if using the ruler
+|None
 |35Gi
 |83Gi
 |171Gi
 endif::restricted[]
 
 |Total disk requests
+|40Gi
 |430Gi
 |430Gi
 |590Gi
 
 ifdef::restricted[]
 |Total disk requests if using the ruler
-|650Gi
-|650Gi
+|80Gi
```

Comment:
@abrennan89 I am not able to see the content for the restricted conditionals in the preview. Just curious, where do I see it, and how? Thanks!

Reply:
It's included in the docs here: https://69602--ocpdocs-pr.netlify.app/openshift-enterprise/latest/network_observability/installing-operators#loki-deployment-sizing_network_observability

Reply:
It doesn't look like it's showing up which is odd, since I believe that it used to. Let me check on this.

Reply:
AH, brilliant catch. It's because the filename that the content is not restricted in has been changed. I'll update it today. Thanks so much!